bootstrap.py

```diff
@@ -90,10 +164,19 @@ def bootstrap(args):
 
     # Clone dependencies
     for dependency, specifier in DEPENDENCIES.items():
-        package = specifier["git"]
-        version = specifier["version"]
+        eprint(f"dep \"{dependency}\" = \"{specifier}\"")
         path = os.path.join(SLM_DEPS_DIR, dependency)
-        fetch_package_into(path, package, version)
+        if "git" in specifier:
+            package = specifier["git"]
+            version = specifier["version"]
+            fetch_package_into(path, package, version)
+        elif "pkg" in specifier and "type" in specifier and specifier["type"]=="conan":
+            package = specifier["pkg"]
+            version = specifier["version"]
+            options = specifier["options"]
+            download_conan_package_into(path, package, version, options)
+        else:
+            raise Exception(f"unknown dependency type: \"{dependency}\" = \"{specifier}\"")
```

Comment:
I think this file has generally been using the `error()` function for indicating faults.

Reply:
So, in general, I would favor raising exceptions that can be caught by the caller rather than a hard exit.
In this case, it doesn't matter much, so I'll update for consistency.